draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -0,0 +1,1568 @@
+     2.2.  ACTN for Network Slicing  . . . . . . . . . . . . . . . .   5
+   3.  Model Applicability . . . . . . . . . . . . . . . . . . . . .   5
+   4.  YANG Model Overview . . . . . . . . . . . . . . . . . . . . .   8
+   5.  Model Tree Structure  . . . . . . . . . . . . . . . . . . . .   8
+   6.  YANG Modules  . . . . . . . . . . . . . . . . . . . . . . . .  11
+   7.  Manageability Considerations  . . . . . . . . . . . . . . . .  18
+   8.  Security Considerations . . . . . . . . . . . . . . . . . . .  19
+   9.  IANA Considerations . . . . . . . . . . . . . . . . . . . . .  19
+   10. References  . . . . . . . . . . . . . . . . . . . . . . . . .  20
+     10.1.  Normative References . . . . . . . . . . . . . . . . . .  20
+     10.2.  Informative References . . . . . . . . . . . . . . . . .  21
+   Appendix A.  Acknowledgments  . . . . . . . . . . . . . . . . . .  22
+   Appendix B.  Data Tree for the Example in Section 3.1 . . . . . .  22
+     B.1.  Native Topology . . . . . . . . . . . . . . . . . . . . .  23
+   Authors' Addresses  . . . . . . . . . . . . . . . . . . . . . . .  27
+
+
+
+
+
+
+
+Liu, et al.              Expires 13 August 2023                 [Page 2]
+
+Internet-Draft      Network Slice Topology Data Model      February 2023
+
+
+1.  Introduction
+
+   This document defines a YANG [RFC7950] data model for representing,
+   managing, and controlling IETF network slices as abstract network
+   topologies, where the network slices are defined in
+   [I-D.ietf-teas-ietf-network-slices].
+
+   The defined data model is an interface between customers and
+   providers for configurations and state retrievals, so as to support
+   network slicing as a service.  Through this model, a customer can
+   learn the slicing capabilities and the available resources of the
+   provider.  A customer can request or negotiate with a network slicing
+   provider to create an instance.  The customer can incrementally
+   update its requirements on individual topology elements in the slice
+   instance, and retrieve the operational states of these elements.
+   With the help of other mechanisms and data models defined in IETF,
+   the telemetry information can be published to the customer.
+
+   The YANG model defines technology-agnostic constructs common to
+   network slicing at network layers of different technologies, e.g.
+   IP/MPLS(-TP), OTN and WDM.  Therefore, this model may be used as a
+   common base model on which other network slicing models, such as
+   [I-D.ietf-ccamp-yang-otn-slicing], may augments with technology-
+   specific constructs.
+
+   As described in Section 3 of
+   [I-D.contreras-teas-slice-controller-models], the data model defined
+   in this document complements the data model defined in
+   [I-D.ietf-teas-ietf-network-slice-nbi-yang].  In addition to the
+   provider's view, the data model defined in this document models the
+   Type 2 service defined in [RFC8453].
```

Comment:
We need to elaborate the text to include in [I-D.contreras-teas-slice-controller-models]

Reply:
Changes need for I-D.contreras-teas-slice-controller-models wrt the updated context of draft-liu.